_posts/2021-04-14-accessing-the-relay-store.markdown

```diff
@@ -0,0 +1,260 @@
+---
+layout: epic
+title: "A Cool Way to Access the Relay Store"
+date: 2021-04-14
+categories: [relay, graphql, react, redis, tooling]
+author: anna
+---
+
+Need to access the Relay store without using a mutation? Relay's `commitLocalUpdate` allows you to modify the relay
```

Comment:
```suggestion
Need to access the Relay store without using a mutation? Relay's `commitLocalUpdate` allows you to modify the Relay
```

Maybe this isn't the right kind of feedback at this time, but there's some inconsistency in capitalization of Relay. There are a few other lowercase ones elsewhere.

Reply:
This is helpful!